src/registry_schemas/example_data/schema_data.py

```diff
@@ -1368,7 +1368,7 @@
             'foundingDate': '2018-01-01T00:00:00+00:00',
             'identifier': 'U1234567',
             'legalName': 'legal name - Test',
-            'legalType': 'ULC'
```

Comment:
There will still be ULC, CCC, etc.

Reply:
I will add the big list of types back in so that these will still pass validation